apps/test/unit/code-studio/pd/form_components/FormControllerTest.js

```diff
@@ -242,7 +243,7 @@ describe('FormController', () => {
         stubRequiedFields(['included', 'excluded']);
         DummyPage1.associatedFields = ['included'];
 
-        const validated = form.getNode().validateCurrentPageRequiredFields();
+        const validated = form.instance().validateCurrentPageRequiredFields();
```

Comment:
Under enzyme 3, when rendering with `mount`,  [`getNode()` should be replaced with
`instance()`](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#with-mount-getnode-should-not-be-used-instance-does-what-it-used-to).